From 3e94b080801219b7c4c0608bde69f942d11b8fdb Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Mon, 27 May 2019 15:19:16 +0900
Subject: [PATCH 5/9] spp_pcap: rename define of buffer size for cmds

For refactoring, rename SPP_CMD_NAME_BUFSZ and SPP_CMD_NAME_VALUE_BUFSZ
to SPPWK_NAME_BUFSZ and SPPWK_VAL_BUFSZ to be more simple and specific.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 src/pcap/cmd_parser.c | 2 +-
 src/pcap/cmd_parser.h | 8 ++++----
 src/pcap/cmd_runner.c | 4 ++--
 3 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/pcap/cmd_parser.c b/src/pcap/cmd_parser.c
index b91b39c..a5f3a4a 100644
--- a/src/pcap/cmd_parser.c
+++ b/src/pcap/cmd_parser.c
@@ -92,7 +92,7 @@ parse_command_in_list(struct spp_command_request *request,
 	int i = 0;
 	int argc = 0;
 	char *argv[SPPWK_MAX_PARAMS];
-	char tmp_str[SPPWK_MAX_PARAMS * SPP_CMD_VALUE_BUFSZ];
+	char tmp_str[SPPWK_MAX_PARAMS * SPPWK_VAL_BUFSZ];
 	memset(argv, 0x00, sizeof(argv));
 	memset(tmp_str, 0x00, sizeof(tmp_str));
 
diff --git a/src/pcap/cmd_parser.h b/src/pcap/cmd_parser.h
index 109b649..5945e62 100644
--- a/src/pcap/cmd_parser.h
+++ b/src/pcap/cmd_parser.h
@@ -21,10 +21,10 @@
 #define SPPWK_MAX_PARAMS 8
 
 /** command name string buffer size (include null char) */
-#define SPP_CMD_NAME_BUFSZ  32
+#define SPPWK_NAME_BUFSZ  32
 
 /** command value string buffer size (include null char) */
-#define SPP_CMD_VALUE_BUFSZ 111
+#define SPPWK_VAL_BUFSZ 111
 
 /** parse error code */
 enum sppwk_parse_error_code {
@@ -82,8 +82,8 @@ struct spp_command_request {
 /** parse error information */
 struct spp_command_parse_error {
 	int code;                            /**< Error code */
-	char value_name[SPP_CMD_NAME_BUFSZ]; /**< Error value name */
-	char value[SPP_CMD_VALUE_BUFSZ];     /**< Error value */
+	char value_name[SPPWK_NAME_BUFSZ]; /**< Error value name */
+	char value[SPPWK_VAL_BUFSZ];     /**< Error value */
 };
 
 /**
diff --git a/src/pcap/cmd_runner.c b/src/pcap/cmd_runner.c
index 0ce5d49..d5ae3a5 100644
--- a/src/pcap/cmd_runner.c
+++ b/src/pcap/cmd_runner.c
@@ -42,7 +42,7 @@ struct command_result {
 	int code;
 
 	/* Response message */
-	char msg[SPP_CMD_NAME_BUFSZ];
+	char msg[SPPWK_NAME_BUFSZ];
 
 	/* Detailed response message */
 	char error_message[CMD_RES_ERR_MSG_SIZE];
@@ -51,7 +51,7 @@ struct command_result {
 /* command response list control structure */
 struct command_response_list {
 	/* JSON Tag name */
-	char tag_name[SPP_CMD_NAME_BUFSZ];
+	char tag_name[SPPWK_NAME_BUFSZ];
 
 	/* Pointer to handling function */
 	int (*func)(const char *name, char **output, void *tmp);
-- 
2.17.1

